students/stuartkershaw/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,31 @@
+def exchange_first_last(seq):
+    seq[0], seq[-1] = seq[-1], seq[0]
+    return seq
+
+def remove_nth_even(seq):
+    return seq[::2]
+
+def return_specified(seq):
+    return seq[4:-4:2]
+
+def reverse_sequence(seq):
+    return seq[::-1]
+
+def return_specified_thirds(seq):
+    index = int(len(seq) / 3)
+    seg_end = seq[:index]
+    seg_start = seq[index:]
+    return seg_start + seg_end
```

Comment:
I'm assuming the assert fails right? (it looks like it's missing the mid third.)